telemetry/definitions/commonDefinitions.json

```diff
@@ -30,11 +30,26 @@
             ],
             "description": "AWS filetype kind"
         },
+        {
+            "name": "cloudWatchLogsPresentation",
+            "allowedValues": ["ui", "text"],
+            "description": "Presentation mode used in a CloudWatch Logs operation"
```

Comment:
I believe this only applies to when we are viewing any cloudwatch logs resource. What do you think about updating the description to indicate this?

Reply:
I generalized that notion to "presentation mode", in case we wind up with a future metric that can leverage this type.